src/main/java/marquez/client/models/Dataset.java

```diff
@@ -42,6 +42,7 @@
   @Getter @NonNull private final Instant createdAt;
   @Getter @NonNull private final Instant updatedAt;
   @Getter @NonNull private final String sourceName;
+  @Getter @Nullable private final Instant lastModified;
```

Comment:
Given that `Dataset.lastModified` is nullable, we'll want to override the getter method and wrap the value in a `Optional`:

```java
  public Optional<Instant> getLastModified() {
    return Optional.ofNullable(lastModified);
  }
```

Reply:
Done